LICENSE

```diff
@@ -1,6 +1,6 @@
-MIT License
+MIT NON-AI License
```

Comment:
I wonder what's the point of changing the license to NON-AI?

In fact, there is nothing special in the code that could be hidden from the machine. And what could not be used to train her.

Why then change the license to a more strict one?

Reply:
How do you know it’s a her. 

Well, just trying to lower the footprint for readme, commit messages, our issues discussion and the code to not end point up on some system we don’t know about. 
More like a protection for us, if the AI goes wrong, it weren’t us who helped train it.

Reply:
OK